pages/index.js

```diff
@@ -384,7 +384,7 @@ const Page = () => (
             icon="sam"
             color="blue"
             name="Free Zoom Pro"
-            desc="A paid Zoom subscription for your club to run online meetings without the time limit."
+            desc="24/7 access to Zoom Pro enabled meeting rooms for your club (that means no time limit)."
```

Comment:
Thoughts on this?

Or how about:

```suggestion
            desc="24/7 access to Zoom Pro enabled meeting rooms for your club (finally, no time limit)."
```

Reply:
I kinda prefer the first one ("that means no time limit")